src/test/java/net/sf/jabref/logic/importer/fileformat/BibtexParserTest.java

```diff
@@ -127,15 +127,6 @@ public void singleFromStringReturnsNullIfNoEntryRecognized() {
     }
 
     @Test
-    public void parseTwoTimesReturnsSameResult() throws IOException {
```

Comment:
No not really. The code no longer stores the previous result but parses the reader again and again.